File: Python/Gate-Three/task_two.py
def largest_array_index(numbers):

    largest = 0
    array_position = 0

    for outterloop in range(len(numbers)):

        for innerloop in range(outterloop):

            if innerloop > largest :

                largest = innerloop

            if outterloop > array_position:

               array_position = outterloop    

               result = largest , array_position

            return result






#def count_numbers_of_specific_letters_in_index():
#
#characters = "AaaBBc"
#
#counter = 0
#
#
#for index in range(len(characters)):
#
#    for letters in characters:
#
#        if letters == "A":
#
#            counter += 1
#
#            return counter
#
#
#
#

File: Java/Gate-Three/TaskTwo.java
public class TaskTwo{


    public static String CountLettersInString(String characters){


       int counter = 0;

    for(int index = 0; index <= characters.length(); index++){

           for(int letters = 0; letters <= characters.lenth(); letters ++){

                char character = characters.charAt(letters);

                if(character == 'A'){

                    counter = character;

                    counter += 1;
    }
           }
                }
                   return characters;     
    
}




    public static double LargestArrayIndex(double [] [] numbers){

    double largest = 0;


    for(double outterLoop = 0; outterLoop < numbers.length; outterLoop++){

           for(double innerLoop = 0; innerLoop < numbers.length; innerLoop++){

                    if(numbers[outterLoop][innerLoop] > largest){

                       largest = numbers[outterLoop][innerLoop]

        }
           }
               }


            return largest;
}





}


